content-src/experiments/tracking-protection.yaml

```diff
@@ -86,6 +86,7 @@ contributors:
     display_name: 'Rebecca Billings'
     title: 'Senior QA Engineer'
     avatar: /static/images/experiments/tracking-protection/avatars/rebecca-billings.jpg
+contributors_extra: 'This experiment is based on Firefox Tracking Protection technology built by Mozilla employees and contributors over the past several years. Learn more about <a href="https://mzl.la/2fYneDq" target="blank">Firefox Tracking Protection</a>.'
```

Comment:
nit: I'm a huge jerk, so I'll also point out that it's `"_blank"`, not `"blank"`.

Somebody smarter than me (pretty much everybody reading this), should probably also skim https://dev.to/ben/the-targetblank-vulnerability-by-example and/or https://mathiasbynens.github.io/rel-noopener/ and see if we should be doing something extra w/ `rel="noopener"` or whatever.